sdk/cosmosdb/ci.yml

```diff
@@ -0,0 +1,21 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
+
+trigger: none
+
+pr:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - sdk/cosmosdb/cosmos/
```

Comment:
Why wouldn't these path filters be `sdk/<service>`? I'd expect them to be run for anything under this service directory. I realize you still have plans to figure out how to build everything so having a single package to build for now seems fine.

Reply:
Yep - fixed - the template reference was wrong too.